### ISSUE TITLE
non-zero exit status on kaggle, can't install dependent packages

### ISSUE THREAD

**Issue by ChrisRichardMiles** [r-lib/rlang#969]
I run this in a kaggle r kernal: 
`install.packages("rlang", lib="/kaggle/working")`

and I get this output: 
"""
Warning message in install.packages("rlang", lib = "/kaggle/working"):
“installation of package ‘rlang’ had non-zero exit status”
""""

I'm trying to use packages that depend on rlang and they always stop their installation and tell me something about rlang and a non-zero exit status. 

How can we get rlang to work on kaggle? 
Thank you very much.

**Comment by lionel-** [comment]
Try `type = "binary"`.

**Comment by lionel-** [comment]
If this solves it (please let me know) you can also set `options(pkgType = "binary")` in your RProfile to always install from binary from now on, which I recommend on Windows and macOS.

Also was this all the output that you got or just the end?

**Comment by ChrisRichardMiles** [comment]
@lionel- 
I tried `install.packages("rlang", lib="/kaggle/working", type='binary')` and I got 

`Error in install.packages("rlang", lib = "/kaggle/working", type = "binary"): type 'binary' is not supported on this platform
Traceback:

1. install.packages("rlang", lib = "/kaggle/working", type = "binary")
2. stop("type 'binary' is not supported on this platform")`

Maybe it is just a problem with kaggle.com. 

> Also was this all the output that you got or just the end?
This was all the output that I got.
